bin/all_sky_search/pycbc_fit_sngls_over_multiparam

```diff
@@ -93,27 +157,16 @@ for param, slog in zip(args.fit_param, args.log_param):
     else:
         raise ValueError("invalid log param argument, use 'true', or 'false'")
 
-if 'count_in_template' in fits.keys(): # older files may not have this dataset
```

Comment:
As we don't really need the backwards compatibility to before this was put in, these checks for whether the `count_in_template` is set isn't really needed anymore